pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -364,9 +364,10 @@ func (e InstanceTypeFilterError) Error() string {
 }
 
 //nolint:gocyclo
-func filterInstanceTypesByRequirements(instanceTypes []*cloudprovider.InstanceType, requirements scheduling.Requirements, podRequests, daemonRequests, totalRequests corev1.ResourceList) (cloudprovider.InstanceTypes, error) {
+func filterInstanceTypesByRequirements(instanceTypes []*cloudprovider.InstanceType, requirements scheduling.Requirements, podRequests, daemonRequests, totalRequests corev1.ResourceList, autoRelaxMinValues bool) (cloudprovider.InstanceTypes, bool, error) {
```

Comment:
What's the behavior in this filter function if you have another pod that comes in that doesn't cause us to relax the minValues further and can fit on the NodeClaim? Will we allow it to schedule? In my mind we should so long as it doesn't cause us to continue to relax our NodeClaim even further